src/create_disk.sh

```diff
@@ -26,6 +26,8 @@ grub_script="$1" && shift
 os_name="$1" && shift
 extrakargs="$1" && shift
 
+set -x
```

Comment:
Might as well just shove it as part of the `set -euo pipefail` bit? That way we'll see the expansion of all the args.

Reply:
I thought about that but the argument parsing feels...uninteresting.  Most of those args we'll see down below in the command invocations.